.travis.yml

```diff
@@ -1,49 +1,59 @@
 language: php
-sudo: required
-dist: precise
+sudo: true
```

Comment:
If you can figure out how to use the [`apt` addon](https://docs.travis-ci.com/user/installing-dependencies/#Installing-Packages-with-the-APT-Addon), we can remove `sudo`